object/service.proto

```diff
@@ -19,6 +19,16 @@ service ObjectService {
   // be restored by concatenation of object message payload and all chunks
   // keeping receiving order.
   //
+  // Extended headers can change `Get` behaviour:
+  // * __NEOFS__NETMAP_EPOCH \
+  //   Will use the requsted version of Network Map for object placement
+  //   calculation.
+  // * __NEOFS__NETMAP_LOOKUP_DEPTH \
+  //   Will try older versions of network map to find an object until the depth
```

Comment:
I'd clarify:
```suggestion
  //   Will try older versions (starting from `__NEOFS__NETMAP_EPOCH` if specified or the latest one otherwise) of network map to find an object until the depth
```
in order to prevent infinite request loop if storage nodes always start processing from the latest epoch.